<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoryDialog</class>
 <widget class="QDialog" name="CategoryDialog">
  <property name="windowTitle"><string>Categories</string></property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QListWidget" name="listCategories"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item><widget class="QLineEdit" name="editName" placeholderText="Category name"/></item>
     <item><widget class="QComboBox" name="comboType"><item><property name="text"><string>expense</string></property></item><item><property name="text"><string>income</string></property></item></comboType></item>
     <item><widget class="QPushButton" name="btnAdd" text="Add"/></item>
    </layout>
   </item>
  </layout>
 </widget>
</ui>
